test/oidc-test-provider.ts

```diff
@@ -227,7 +227,11 @@ async function spawnBrowser(
           ...options.capabilities,
           'goog:chromeOptions': {
             binary: electronPath,
-            args: [`--app=${url}`, '--'],
+            args: [`--app=${url}`, '--disable-save-password-bubble', '--'],
+            prefs: {
+              'profile.password_manager_enabled': false,
+              credentials_enable_service: false,
+            },
```

Comment:
Pulled this in from the node update pr. This prevents the popups about passwords that get in the way of local testing.